### PR TITLE
docs: add zulu api endpoints

### PR DESCRIPTION
**Pull request by junbeomlee**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,3 +2,137 @@
...
+        {
+          "value": "38089899992",
+          "decimal": 8,
+          "symbol": "BTC"
+        }
+      ],
+      "data": {}
+    }
+  ]
+  ```
```
Comment (baumstern) [inline docs/README.md:120]: L84:100이랑 중복입니다!

Comment (hea9549) [reply docs/README.md:120]: /{id} 빠진듯해여~

Comment (hihiboss) [reply docs/README.md:120]: L84의 URI가 `/networks/{networkId}/wallets/{walletId}`면 될듯요!

File: docs/README.md [hunk 2]
```diff
@@ -2,3 +2,137 @@
...
+  Create a transfer
+  ```
+  {
+    "to": "0x3a065000ab4183c6bf581dc1e55a605455fc6d61",
+    "amount": "3"
+    "data": {}
+  }
+  
+  {
+    "hash": "0x3a065000ab4183c6bf581dc1e55a605455fc6d61",
```
Comment (hihiboss) [inline docs/README.md:135]: 요거는 tx hash인가여??

Comment (junbeomlee) [reply docs/README.md:135]: tx hash 맞음!

File: docs/README.md [hunk 3]
```diff
@@ -2,3 +2,137 @@
 
 ![Architecture](./Architecture.png)
 
+
+
+## API
+
+### Network
```
Comment (zeroFruit) [inline docs/README.md:9]: **suggestion**: how about change the API docs format?

I suggest following format: https://gist.github.com/iros/3426278

**suggestion**: change API header from h2(##) to h1(##), and network from h3(###) to h2(##)

Comment (junbeomlee) [reply docs/README.md:9]: That sounds good!

File: docs/README.md [hunk 4]
```diff
@@ -2,3 +2,137 @@
...
+
+- [GET] /networks
+
+  Get the list of supported blockchain networks
+
+  ```
+  {
+    [
+      {
+        "id": "988f08b3b21e37269dcc2bbe979c9e69308c1c6b96dc220ce6635a2d728bb31e"
```
Comment (boohyunsik) [inline docs/README.md:19]: What `id` means?

**Comment by junbeomlee** [comment]
We will close the endpoint-related issues as we need more discussion.

**Pull request closed without merging**